app/rest/connector-generator/src/test/java/io/syndesis/connector/generator/swagger/BaseSwaggerGeneratorExampleTest.java

```diff
@@ -39,7 +39,7 @@
     private final String specification;
 
     public BaseSwaggerGeneratorExampleTest(final String connectorQualifier, final String name) throws IOException {
-        specification = resource("/swagger/" + name + ".swagger.json");
+        specification = resource("/swagger/" + name + ".swagger.json", "/swagger/" + name + ".swagger.yaml");
```

Comment:
Supporting YAML definition is separate to this PR, but I'm OK to include it here or have it in separate PR - me likey granular PRs :)

Reply:
At least in a separate commit anyway please.

Reply:
We already parse Swagger files using the Swagger API that supports both JSON and YAML, so this doesn't provide additional functionality. It was just somewhat easier to craft a YAML Swagger file for this test.

Reply:
Ah OK.